src/deploy-cromwell-on-azure/Utility.cs

```diff
@@ -2,13 +2,68 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Security.Cryptography;
 using System.Text.RegularExpressions;
 
 namespace CromwellOnAzureDeployer
 {
     public static class Utility
     {
+        public static string GetFileContent(params string[] pathComponentsRelativeToAppBase)
+        {
+            var embeddedResourceName = $"deploy-cromwell-on-azure.{string.Join(".", pathComponentsRelativeToAppBase)}";
+            var embeddedResourceStream = typeof(Deployer).Assembly.GetManifestResourceStream(embeddedResourceName);
+
+            using var reader = new StreamReader(embeddedResourceStream);
+            return reader.ReadToEnd().Replace("\r\n", "\n");
+        }
+
+        public static string UpdateExistingSettingsFileContentV300(string existingSettingsFileContent)
+        {
+            var existingSettingsFileContentLines = existingSettingsFileContent.Replace("\r\n", "\n").Split('\n', StringSplitOptions.RemoveEmptyEntries);
```

Comment:
There are already private methods DelimitedTextToDictionary and DictionaryToDelimitedText - see method PatchAccountNamesFileV240Async, that make updating the settings files easier. Maybe move those to Utility.